pkg/maintainer/relay.go

```diff
@@ -0,0 +1,85 @@
+package maintainer
+
+import (
+	"fmt"
+
+	"github.com/keep-network/keep-common/pkg/chain/ethereum"
+	"github.com/keep-network/keep-core/pkg/bitcoin"
+	"github.com/keep-network/keep-core/pkg/tbtc"
+	"github.com/mitchellh/mapstructure"
+	"github.com/spf13/viper"
+)
+
+type Config struct {
```

Comment:
I'd move the config-related stuff to a separate file `pkg/maintainer/config.go` and cover the functions with unit tests.

Reply:
I moved the config to `pkg/maintainer/config.go`, but I didn't add any unit tests, since I simplified the config and now it's a simple structure without any methods.